contracts/pool/src/tests/pool_deployment.rs

```diff
@@ -0,0 +1,133 @@
+extern crate std;
+use phoenix::utils::{StakeInitInfo, TokenInitInfo};
+use soroban_sdk::{testutils::Address as _, Address, Env};
+
+use super::setup::{deploy_liquidity_pool_contract, deploy_token_contract};
+use crate::{
+    stake_contract,
+    storage::{Config, PairType},
+};
+
+use crate::contract::{LiquidityPool, LiquidityPoolClient};
+use crate::tests::setup::{install_stake_wasm, install_token_wasm};
+
+#[test]
+fn confirm_pool_contract_deployment() {
```

Comment:
Purpose of that test was to check if stake contract is deployed correctly.
Since you changed it it doesn't matter now, but just saying.

Reply:
that's brought to the old state